lib/remap-lang/src/expression/variable.rs

```diff
@@ -40,30 +28,21 @@ impl Expression for Variable {
     fn execute(&self, state: &mut state::Program, _: &mut dyn Object) -> Result<Value> {
         let mut value = state.variable(&self.ident).cloned().unwrap_or(Value::Null);
 
-        if let Some(path) = &self.path {
-            return path.execute(state, &mut value).map_err(|err| {
-                let err = match err {
-                    E::Expression(ExprErr::Path(err)) => Error::Query(err),
-                    _ => Error::Unknown(err.to_string()),
-                };
-
-                ExprErr::Variable(self.ident.clone(), err).into()
-            });
+        match &self.path {
+            Some(path) => Ok(path.execute(state, &mut value).ok().unwrap_or(Value::Null)),
+            None => Ok(value),
         }
-
-        Ok(value)
     }
 
     fn type_def(&self, state: &state::Compiler) -> TypeDef {
         state
             .variable_type(&self.ident)
             .cloned()
-            // TODO: we can make it so this can never happen, by making it a
```

Comment:
Could we open an issue to track this? I do think it'd be nice to be able to catch typo'd variable names.